tests/queries/0_stateless/00826_cross_to_inner_join.sql

```diff
@@ -1,6 +1,7 @@
 SET enable_optimize_predicate_expression = 0;
 SET optimize_move_to_prewhere = 1;
 SET convert_query_to_cnf = 0;
+SET reorder_joins = 0;
```

Comment:
Is the result for queries in this tests compatible with current behaviour?

Reply:
Basically yes, although some queries here are underspecified. They return multiple rows but lack an ORDER BY. For example: `SELECT * FROM t1_00826, t2_00826 where t1_00826.a = t2_00826.a;` can get a different ordering of rows if join reordering is enabled.

The queries that do `EXPLAIN SYNTAX SELECT ..` will have different results if the tables are reordered.

I copy pasted this file as a starting point for `00826_cross_to_inner_join_with_reordering.sql`, which has `SET reorder_joins=1;`, so mostly the same queries are repeated there (with the necessary ORDER BYs added, and EXPLAIN SYNTAX queries removed).

I decided to add `SET reorder_joins=0;` here, so this can keep testing the same path it tested earlier.